osbs/api.py

```diff
@@ -643,6 +655,13 @@ def create_orchestrator_build(self, **kwargs):
         if not self.can_orchestrate():
             raise OsbsValidationException("can't create orchestrate build "
                                           "when can_orchestrate isn't enabled")
+        extra = set()
+        for unwanted in ('platform', 'release'):
```

Comment:
I agree with this change, but it seems unrelated to node selectors. Can we split up into different commits?